remoting/tests/src/test/resources/org/jboss/as/remoting/remoting-with-outbound-connection-group.xml

```diff
@@ -0,0 +1,12 @@
+<subsystem xmlns="urn:jboss:domain:remoting:3.1">
+    <connector name="test-connector" socket-binding="test"/>
+    <outbound-connections>
+        <remote-outbound-connection-group name="group" outbound-socket-binding-refs="dummy-outbound-socket, other-outbound-socket">
```

Comment:
This is defined in xsd as xs:list which is *whitespace* delimited, so why the commas...

http://www.w3schools.com/schema/el_list.asp